tests/test_templater.py

```diff
@@ -86,7 +86,8 @@ def test_set_template_dryrun(): #pylint: disable=unused-variable
         templater.set_template(args)
     result = outstring.getvalue()
 
-    assert result == outcome
+    for outcome_line in outcome.split('\n'):
+        assert outcome_line in result
```

Comment:
This change in how the tests are validated allows us to keep adding log lines to output without updating verbatim the output in the test. 

Let me know if you think this requirement is too loose.

Reply:
Its not a strict match check so it should be a-ok, and we do that for similar reasons in other places.
Hm, now that you mention it, we should be checking specifically for the log.info() messages from --dry_run rather than the block text. Maybe that should be part of a later general refactoring pass?

Reply:
From one of our sessions this week, it may be that we want to do a group walk through of the tests in general to make sure we've tested everything appropriately, and to organize the tests more effectively.